runners/direct-java/src/main/java/org/apache/beam/runners/direct/DirectRunner.java

```diff
@@ -51,8 +51,8 @@
 import org.joda.time.Duration;
 
 /**
- * An In-Memory implementation of the Dataflow Programming Model. Supports Unbounded
- * {@link PCollection PCollections}.
+ * A {@link PipelineRunner} that executes a {@link Pipeline} within the process that constructed the
+ * {@link Pipeline}.
```

Comment:
Actually some flavor text in follow-up paragraph would be good since the DirectRunner is a huge deal for users. Something along the lines of "use this to get your logic right, run unit tests, and execute optional behavioral enforcements on your pipeline".